src/main/java/ch/njol/skript/FeatureConfig.java

```diff
@@ -0,0 +1,144 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+import ch.njol.skript.config.Config;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.config.OptionSection;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.util.FileUtils;
+import ch.njol.util.Pair;
+
+public class FeatureConfig {
+
+	public final static OptionSection alterSyntax = new OptionSection("alterSyntax");
+	public final static OptionSection features = new OptionSection("features");
+
+	private static Map<String, String> alteredPatterns = new HashMap<>();
+	private static List<String> disabledClassNames = new ArrayList<>();
+	private static List<String> disabledPatterns = new ArrayList<>();
+
+	private static boolean loaded;
+
+	public static Pair<Boolean, String[]> contains(String className, String... patterns) {
```

Comment:
Is there a reason for using a pair here? The second value doesn't seem to be used.